docs/source/using/addons.adoc

```diff
@@ -161,6 +161,23 @@ $ minishift addons enable anyuid
 $ minishift addons disable anyuid
 ----
 
+[[apply-addons]]
+== Applying add-ons
+
+Add-ons can be explicitly executed with the link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
+You can use _add-on_ _apply_ command for both enabled and disabled add-ons. For applying multiple add-ons at once please specify multiple add-on names separated by space.
```

Comment:
I think we can drop "please". I know we want to be polite, but for the docs this is not needed imo. Also, I think we can drop "multiple" from "specify multiple add-on names"